dev/mongosh.ts

```diff
@@ -0,0 +1,230 @@
+// https://docs.mongodb.com/mongodb-shell/reference/options/
+
+import { decodedTextSpanIntersectsWith } from "typescript";
+
+const completionSpec: Fig.Spec = {
+  name: "mongosh",
+  description:
+    "The MongoDB Shell, mongosh , is a fully functional JavaScript and Node.js 14.x REPL environment for interacting with MongoDB deployments",
```

Comment:
```suggestion
    "The MongoDB Shell, mongosh  is a fully functional JavaScript and Node.js 14.x REPL environment for interacting with MongoDB deployments",
```

Reply:
I think that comma is useful there, it's an aside of the phrase.

Reply:
I made a mistake, wanted to remove the space before the comma, not remove the comma itself 😅
Here is the new suggestion:
```suggestion
    "The MongoDB Shell, mongosh, is a fully functional JavaScript and Node.js 14.x REPL environment for interacting with MongoDB deployments",
```